contracts/interchain-token-service/src/msg.rs

```diff
@@ -33,6 +34,13 @@ pub enum ExecuteMsg {
     /// The admin is allowed to remove the ITS address of a chain for emergencies.
     #[permission(Elevated)]
     DeregisterItsContract { chain: ChainNameRaw },
+
+    #[permission(Governance)]
+    SetChainConfig {
```

Comment:
Hmm, how about we define `RegisterChainConfig`, and `UpdateChainItsContract` msg types instead? Register config will allow setting the config, whereas update will only allow updating specific fields. It's similar to what we do in the [router](https://github.com/axelarnetwork/axelar-amplifier/blob/main/packages/router-api/src/msg.rs#L21) where the msg id format isn't allowed to be changed, but the gateway address is

Reply:
Are you suggesting a removal of `DeregisterItsContract`?